.vortex/tooling/tests/Unit/FetchDbLagoonTest.php

```
@@ -14,361 +14,260 @@ class FetchDbLagoonTest extends UnitTestCase {
+        ['backupid' => 'old-id', 'source' => 'mariadb', 'created' => '2024-01-01 00:00:00', 'restored' => 'false', 'restorestatus' => ''],
+        ['backupid' => 'latest-id', 'source' => 'mariadb', 'created' => '2024-01-02 00:00:00', 'restored' => 'false', 'restorestatus' => ''],
+        ['backupid' => 'files-id', 'source' => 'nginx', 'created' => '2024-01-03 00:00:00', 'restored' => 'false', 'restorestatus' => ''],
+      ],
+    ]) ?: '';
+  }
+
+  protected function configCmd(): string {
+    return "'lagoon' config add --force --lagoon 'amazeeio' --graphql 'https://api.lagoon.amazeeio.cloud/graphql' --hostname 'ssh.lagoon.amazeeio.cloud' --port '32222'";
+  }
+
+  protected function lagoonCmd(string $subcommand, bool $with_ssh = TRUE): string {
```

PHPMD (2.15.0)

[error] The method lagoonCmd has a boolean flag argument $with_ssh, which is a certain sign of a Single Responsibility Principle violation. (undefined)

(BooleanArgumentFlag)